tailnet/test/integration/network.go

```diff
@@ -390,33 +390,38 @@ func createFakeInternet(t *testing.T) fakeInternet {
 }
 
 type TriangleNetwork struct {
-	InterClientMTU int
+	Client1MTU int
 }
 
 type fakeTriangleNetwork struct {
-	NamePrefix             string
-	ServerNetNS            *os.File
-	Client1NetNS           *os.File
-	Client2NetNS           *os.File
-	ServerClient1VethPair  vethPair
-	ServerClient2VethPair  vethPair
-	Client1Client2VethPair vethPair
+	NamePrefix      string
+	ServerNetNS     *os.File
+	Client1NetNS    *os.File
+	Client2NetNS    *os.File
+	RouterNetNS     *os.File
+	ServerVethPair  vethPair
+	Client1VethPair vethPair
+	Client2VethPair vethPair
 }
 
-// SetupNetworking creates multiple namespaces with veth pairs between them
-// with the following topology:
+// SetupNetworking creates multiple namespaces with a central router in the following topology
 // .
-// .     ┌────────────────────────────────────────────┐
-// .     │                  Server                    │
-// .     └─────┬───────────────────────────────────┬──┘
-// .           │fdac:38fa:ffff:2::3                │fdac:38fa:ffff:3::3
-// .       veth│                               veth│
-// .           │fdac:38fa:ffff:2::1                │fdac:38fa:ffff:3::2
-// .   ┌───────┴──────┐                      ┌─────┴───────┐
-// .   │              │   fdac:38fa:ffff:1::2│             │
-// .   │  Client 1    ├──────────────────────┤  Client 2   │
-// .   │              │fdac:38fa:ffff:1::1   │             │
-// .   └──────────────┘                      └─────────────┘
+// .   ┌──────────────┐
+// .   │              │
+// .   │  Server      ├─────────────────────────────────────┐
+// .   │              │fdac:38fa:ffff:3::1                  │
+// .   └──────────────┘                                     │ fdac:38fa:ffff:3::2
+// .   ┌──────────────┐                               ┌─────┴───────┐
+// .   │              │            fdac:38fa:ffff:1::2│             │
+// .   │  Client 1    ├───────────────────────────────┤  Router     │
+// .   │              │fdac:38fa:ffff:1::1            │             │
+// .   └──────────────┘                               └─────┬───────┘
+// .   ┌──────────────┐                                     │ fdac:38fa:ffff:2::2
+// .   │              │                                     │
+// .   │  Client 2    ├─────────────────────────────────────┘
+// .   │              │fdac:38fa:ffff:2::1
+// .   └──────────────┘
```

Comment:
Can we change the `::1` addresses to belong to the router instead of the leaf nodes? I get that this doesn't affect the behavior of the test, but in most router setups (especially IPv4) the router typically uses `.1` for itself. Since we're kinda treating these IPv6 addresses like they are IPv4 addresses with sequential numbering, it seems more intuitive to give `::1` to the router.

Reply:
This was something I noticed previously (between clients and server) but decided not to bring up since there wasn't a router.